client/src/components/users/UserAvatarName.vue

```diff
@@ -17,10 +20,17 @@
 <script setup lang="ts">
 import { IonAvatar, IonText } from '@ionic/vue';
 
-defineProps<{
-  userAvatar: string;
-  userName?: string;
-}>();
+withDefaults(
```

Comment:
You shouldn't need the `withDefaults` since `clickable` is marked as `optional`, it will default to undefined and be evaluated as false. Not that is wrong, it just adds a bit of complexity for not much gains.

Reply:
done